packages/types/message.d.ts

```diff
@@ -0,0 +1,24 @@
+export type CumulusMeta = {
+  execution_name: string
+  queueName?: string
+  state_machine: string
+  parentExecutionArn?: string
+  asyncOperationId?: string
+};
+
+export type ReplaceConfig = {
+  Bucket: string
+  Key: string
+  TargetPath?: string
+};
+
+export type CumulusMessage = {
+  cumulus_meta: CumulusMeta
+  meta: object
+  payload: unknown
```

Comment:
There is an optional `error` property as well.

Reply:
I think it is `exception`?

Reply:
Correct you are.